### DISCUSSION TITLE
Pan using Minimap

### DISCUSSION THREAD

**Discussion by sahilprasad** [xyflow/xyflow#1208] [Ideas]
I always catch myself trying to pan using the Minimap, is this something that can be easily supported?

**Comment by fonpah** [comment]
i am faced with thesame issue, so i thought i use a second react-flow-renderer for the it.  which surprisingly worked. Now the issue is how do i sync the main renderer with the minimap or better still how to i pan or select the a node in the maon renderer by interacting with the minimap.  Or How do is scale the main flow to fit into the minimap?
Any ideas?

**Comment by moklick** [comment]
locked in favour of #2166
